src/EnergyPlus/OutputProcessor.cc

```diff
@@ -5617,6 +5617,9 @@ namespace OutputProcessor {
         case OutputProcessor::Unit::W_K:
             return "W/K";
             break;
+        case OutputProcessor::Unit::K_W:
```

Comment:
@JasonGlazer it's converted from enum to string here.

Reply:
Good.